deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1202,31 +1206,72 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
         }
     }
 
+    @available(iOS 14, *)
+    private func clipperButtonMenu() -> UIMenu {
+        func action(localized: String, systemImage: String, attributes: UIMenuElement.Attributes = [], handler: @escaping () -> Void) -> UIAction {
+            UIAction(title: String.localized(localized), image: UIImage(systemName: systemImage), attributes: attributes, handler: { _ in handler() })
+        }
+        var actions = [UIMenuElement]()
+        actions.append(action(localized: "camera", systemImage: "camera", handler: showCameraViewController))
+        let galleryImage = if #available(iOS 16, *) { "photo.stack" } else { "photo" }
+        actions.append(action(localized: "gallery", systemImage: galleryImage, handler: showPhotoVideoLibrary))
+        actions.append(action(localized: "files", systemImage: "folder", handler: showDocumentLibrary))
+        if dcContext.hasWebxdc() {
+            actions.append(action(localized: "webxdc_apps", systemImage: "app", handler: showWebxdcSelector))
```

Comment:
```suggestion
            actions.append(action(localized: "webxdc_apps", systemImage: "square.grid.2x2", handler: showWebxdcSelector))
```

this icon fits better to what desktop and android are using - and even without, i think this looks more like "icons" than just the single square

android:
![Screenshot 2025-01-16 at 14 42 57](https://github.com/user-attachments/assets/b999eb39-bace-4924-acc5-0f60f6bc5132)

desktop:

<img width=150 src=https://github.com/user-attachments/assets/104412f2-6957-4ec9-9aed-2d34800354f9>

iOS with this suggestion:

<img width=250 src=https://github.com/user-attachments/assets/701eac06-9521-4c3c-a057-dfe94c17fc28>